### DISCUSSION TITLE
Unable to access Laravel website

### DISCUSSION THREAD

**Discussion by siguici** [laravel/framework#50799] [General]
Hey Laravel team!

I don't know if the problem is related to our location (Nabeul, Tunisia) but [Laravel website](https://laravel.com) has been down for a while. I thought it was related to my devices but it's the same with my colleagues.

**Comment by crynobone** [comment]
https://downforeveryoneorjustme.com/laravel.com

**Comment by siguici** [comment]
It seems to work however we all have the same error (ERR_TIMED_OUT): laravel.com took too long to respond.

We will try again later with other colleagues to see if the problem persists.
